src/ray/raylet/node_manager.cc

```diff
@@ -1526,6 +1547,14 @@ void NodeManager::SubmitTask(const Task &task, const Lineage &uncommitted_lineag
             RAY_LOG(WARNING) << "A task was resubmitted, so we are ignoring it. This "
                              << "should only happen during reconstruction.";
             TreatTaskAsFailedIfLost(task);
+          } else if (task.GetTaskExecutionSpec().ActorVersion() <
+                     actor_entry->second.GetActorVersion()) {
+            // The task was for an older version of the actor, so do not
+            // execute it. The caller should handle the error once it receives
+            // the latest version notification from the actor table.
```

Comment:
The caller is the worker process that submitted the task? How does it get the version notification from the actor table?